src/components/PhishingWarning.vue

```diff
@@ -64,7 +64,7 @@ export default {
     width: 100%;
     text-align: left;
     padding: 8px;
-    margin-bottom: 0 5px;
+    margin-bottom: 0 calc(var(--default-grid-baseline) * 2);
```

Comment:
```suggestion
    margin: calc(var(--default-grid-baseline) * 2);
```

Reply:
`-bottom` is a leftover mb, But I do believe that the margin should only be top - bottom. don't you thing so? 
so more like `margin: calc(var(--default-grid-baseline) * 2) 0` like this ->
![image](https://github.com/user-attachments/assets/f10e64cc-9419-4b6d-a5f5-6eee37cca4c9)
otherwise it wouldn't it pop out unless that's what you're going for 
<img width="905" alt="image" src="https://github.com/user-attachments/assets/bdbc43f1-8bc2-48ad-a686-36c5386a6dba">

Reply:
I think we're going for something more like in your second screenshot. Pop factor looks very similar to me